trunk/modules/python/dataset/ingest/labeled.py

```diff
@@ -101,6 +101,13 @@ def readImagery() :
     # join the threads and complete
     workQueueData.join()
 
+def mostCommon(stuff, func):
+    '''Find the most common result of func yielded by stuff'''
+    from collections import Counter
+    counter = Counter([func(s) for s in stuff])
+    common = counter.most_common()[0][0] # first tuple, first elem
+    return common
+

```

Comment:
we should check the speed of this versus using scipy.stats.mode or statistics.mode. They should be equivalent.